.gitignore

```diff
@@ -17,6 +17,7 @@ python_modules
 
 # Built files
 public/
+build/
```

Comment:
I'm not rightly sure. It appears that all that happens the embed API build copies stuff from `public/javascript/embed-api/components/` to `build/javascript/embed-api/components`, and this is the only reference to `build/` anywhere in the project that I can find after about 10 minutes of searching. To be honest, I'd be about 80-90% confident we cant just remove the relevant section from the gulpfile, since not even the embed-api templates make reference to it.

For reference, this is the only reference I could find to that directory: https://github.com/googleanalytics/ga-dev-tools/blob/19dcf7c750af8214e5a306fc0f8e2b28bef7bb40/gulpfile.babel.js#L293-L296

Reply:
Ahh, now I remember. The build directory is for compiled versions of the Embed API plugin examples, so we should probably keep them.

Just revert this change and if you ever get conflicts in the build directory just don't worry about them.

Reply:
To be clear, the embed api builds to `public/`, and the HTML templates serve from there. There's an extra step in gulp to copy the files over, but I can't actually see that the files in `build/` are ever used for anything.

Reply:
https://stackoverflow.com/questions/35047260/embed-api-custom-components-uncaught-typeerror-cannot-read-property-viewselec

Reply:
That answer seems problematic, for two reasons:

- We shouldn't be telling people to download and use webpack output for things, especially not webpack output that's essentially ephemeral and specific to this project and not a purpose-build javascript library. There should be absolutely no expectation that build output can be used as some sort of functional drop-in library in people's projects.
- The answer as written is internally inconsistent. The text says to grab files from `build/...`, but the cited script tag sources content from `public/`.

Reply:
Yeah– I'd link to the real source files at https://github.com/googleanalytics/ga-dev-tools/blob/master/src/javascript/embed-api/components/view-selector2.js, and indicate that it's just an example of how to use `gapi.analytics.createComponent` to create a Component.

Because it has a dependency on `javascript-api-utils`, it's probably for the best to let the user resolve that dependency using whatever their own build system is, and direct them to resources on webpack if they're unfamiliar with how to use that.